python/reg_interface/pkg/setup.cfg

```diff
@@ -50,4 +50,6 @@ exclude=build,lib,.tox,third,*.egg,docs,packages,.eggs
 
 [bdist_rpm]
 requires = python >= 2.6
+    reg_utils-rwreg 
```

Comment:
is this something you're removing with the update to the `specTemplate`?

Reply:
fixed